app/build.gradle

```diff
@@ -0,0 +1,59 @@
+apply plugin: 'com.android.application'
+apply plugin: 'me.tatarka.retrolambda'
+
+android {
+    compileSdkVersion 25
+    buildToolsVersion '25.0.2'
+    defaultConfig {
+        applicationId "com.oneshot.rxjavaapp"
```

Comment:
I use oneshot as my working domain for everything, so I'm not sure what you mean